src/main/java/Bundle_pl.properties

```diff
@@ -1,107 +1,108 @@
-dataverse=Dataverse
-newDataverse=Nouveau dataverse
-hostDataverse=Dataverse h\uFFFDte
-dataverses=Dataverses
-passwd=Mot de passe
-dataset=Ensemble de donn\uFFFDes
-datasets=Ensembles de donn\uFFFDes
-newDataset=Nouvel ensemble de donn\uFFFDes
-files=Fichiers
-file=Fichier
-restricted=En acc\uFFFDs r\uFFFDserv\uFFFD
-restrictedaccess=Acc\uFFFDs r\uFFFDserv\uFFFD
-find=Chercher
-search=Recherche
-language=Langue
-unpublished=Non publi\uFFFD
-cancel=Annuler
-ok=ok
-saveChanges=Enregistrer les modifications
-acceptTerms=Accepter
-submit=Soumettre
-signup=S'inscrire
-login=Se connecter
-email=Courriel
-account=Compte
-requiredField=Champ obligatoire
-new=Nouveau
-identifier=Identifiant
-description=Description
-subject=Sujet
-close=Fermer
-preview=Aper\uFFFDu
-continue=Continuer
-name=Nom
-institution=\uFFFDtablissement
-position=Poste
-affiliation=Affiliation
-createDataverse=Cr\uFFFDer un dataverse
-remove=Supprimer
-done=Termin\uFFFD
-editor=Collaborateur
-manager=Gestionnaire
-curator=Curateur
-explore=Explorer
-download=T\uFFFDl\uFFFDcharger
-downloadOriginal=Format original
-downloadArchival=Format d'archivage (.tab)
-deaccession=Retrait
-share=Partager
-link=Lier
-linked=Li\uFFFD
-harvested=Moissonn\uFFFD
-apply=Appliquer
-add=Ajouter
-delete=Supprimer
-yes=Oui
-no=Non
-previous=Pr\uFFFDc\uFFFDdent
-next=Suivant
-first=Premier
-last=Dernier
-more=Plus\u2026
-less=Moins\u2026
-select=S\uFFFDlectionner\u2026
-selectedFiles=Fichiers s\uFFFDlectionn\uFFFDs
-htmlAllowedTitle=Balises HTML permises
-htmlAllowedMsg=Ce champ prend en charge uniquement certaines <span class="text-info popoverHTML">balises HTML</span>.
+dataverse=Kolekcja
+newDataverse=Nowa kolekcja
+hostDataverse=Kolekcja nadrz\u0119dna
+dataverses=Kolekcje
+passwd=Has\u0142o
+dataset=Zbi�r danych
```

Comment:
Jest pewność że te polskie litery wyświetlą się poprawnie? Zauważyłem że w niektórych miejscach jest unicode a w innych ogonki, takie było założenie?

Reply:
Actually **ó** is a part of ISO-8859-1 standard and it can be left as is without specifying its unicode code.
On my local environment it renders fine